modules/virt-customizing-storage-profile.adoc

```diff
@@ -9,6 +9,8 @@
 
 You can specify default parameters by editing the `StorageProfile` object for the provisioner's storage class. These default parameters only apply to the persistent volume claim (PVC) if they are not configured in the `DataVolume` object.
 
+You cannot modify storage class parameters. To make changes, delete and recreate the storage class. You must then reapply any customizations made to the storage profile.
```

Comment:
Please use "re-create" not "recreate" as per IBM Style Guide https://www.ibm.com/docs/en/ibm-style?topic=word-usage

But "reapply" is correct.

I would also expand to at least "You must then reapply any customizations that were previously made to the storage profile." Arguably to "If you modified the storage profile previously, reapply the changes after re-creating the storage class"; this is better style but ignores the fact that someone else might have made the original changes. Your call.